<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="separatorGroup">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Separador</string>
    </property>
    <widget class="QComboBox" name="separatorSelector">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="currentText">
      <string comment=",">,</string>
     </property>
     <item>
      <property name="text">
       <string comment=",">,</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment=";">;</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string comment="-">-</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QPushButton" name="cargarDatosButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>210</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Cargar datos</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="thresholdGroup">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>109</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Threshold</string>
    </property>
    <widget class="QDoubleSpinBox" name="thresholdSelector">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="cargarArchivoGroup">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>19</y>
      <width>651</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Archivo - (CSV, Text o Excel)</string>
    </property>
    <widget class="QPushButton" name="cargarArchivoButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Cargar Archivo</string>
     </property>
    </widget>
    <widget class="QLabel" name="fileNameText">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>30</y>
       <width>391</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>File name: </string>
     </property>
    </widget>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>281</y>
      <width>651</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="generarArbolButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>480</y>
      <width>651</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Generar el Arbol de decision</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
